protos/flyteidl/admin/matchable_resource.proto

```diff
@@ -52,5 +55,7 @@ message MatchingAttributes {
     ClusterResourceAttributes cluster_resource_attributes = 2; 
 
     ExecutionQueueAttributes execution_queue_attributes = 3;
+
+    string cluster_label = 4;
```

Comment:
do we ever anticipate this changing? is it worth creating a separate proto so we can future-proof this attribute type if we ever want to add or update the way we specify a cluster? (e.g. a list of labels)

Reply:
I explicitly wanted to make sure we do not need a list.

But I see benefit of a separate proto. Will create one.